src/Library/demos/Check Button/main.blp

```diff
@@ -0,0 +1,86 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: _("Check Button");
+  description: _("Allow users to control a binary operation or make a selection from a set of options");
+
+  Box{
+    orientation: vertical;
+    spacing:12;
+
+    Box {
+      halign: center;
+      spacing: 60;
+      Box {
+        orientation: vertical;
+        spacing: 12;
+        Label {
+          label: _("Checkboxes");
+          styles ["title-4"]
+        }
+        Box {
+          orientation: vertical;
+          halign: center;
+          spacing:12;
+          CheckButton checkbox1{
+            label: _("Additions");
+          }
+
+          CheckButton checkbox2{
+            label: "Tomato";
```

Comment:
Make sure all these get marked for translation.

Reply:
Sure. And also if we are not doing the inconsistent button, I'll choose more fitting labels?

Reply:
That sounds good, maybe something that contrasts with the radio buttons being "one at a time".